client/src/main/java/org/apache/celeborn/client/ShuffleClientImpl.java

```diff
@@ -140,11 +140,13 @@ public ShuffleClientImpl(CelebornConf conf, UserIdentifier userIdentifier) {
         new TransportContext(dataTransportConf, new BaseMessageHandler(), true);
     dataClientFactory = context.createClientFactory();
 
-    int retryThreadNum = CelebornConf.pushDataRetryThreadNum(conf);
-    pushDataRetryPool = ThreadUtils.newDaemonCachedThreadPool("Retry-Sender", retryThreadNum, 60);
+    int pushDataRetryThreadNum = conf.pushDataRetryThreadNum();
+    pushDataRetryPool =
+        ThreadUtils.newDaemonCachedThreadPool("Retry-Sender", pushDataRetryThreadNum, 60);
```

Comment:
we'd better add a `celeborn-` or `clb-` prefix to the name of the thread created by celeborn, and please naming in lowercase

Reply:
done